common/src/main/java/com/wynntils/core/components/Models.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright © Wynntils 2022-2024.
+ * Copyright © Wynntils 2022-2025.
```

Comment:
This is a bit unfortunate. This happens when you at some point had changes to a file, but then removed them. I don't think this can be fixed by any other means than rebasing without these changes. And rebasing an open PR is not good either. I suggest continuing the discussion about this PR here, and when the code is ready to merge, then rebase it to remove these spurious changes, or just creating a new PR with the final changes minus these.